projects/js-packages/publicize-components/src/utils/types.ts

```diff
@@ -0,0 +1,17 @@
+export type SIGSettings = {
+	template?: string;
```

Comment:
Should the SIG settings also have the image to use?

Reply:
Updated.